Sources/AppStoreConnectCLI/Model/BetaGroup.swift

```diff
@@ -14,6 +15,7 @@ struct BetaGroup: TableInfoProvider, ResultRenderable, Equatable {
     let publicLinkLimit: Int?
     let publicLinkLimitEnabled: Bool?
     let creationDate: String?
+    var testers: String? // CSV file path
```

Comment:
This is where things can get interesting.. Having variables that will populated when pulled from disk but not populated when retrieved from the network